src/Components/Onboarding/ItemLink.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react"
+import styled, { StyledFunction } from "styled-components"
+
+import { fadeIn, fadeOut } from "../../Assets/Animations"
+import * as fonts from "../../Assets/Fonts"
+
+import Icon from "../Icon"
+
+const anchor: StyledFunction<State & React.HTMLProps<HTMLInputElement>> = styled.a
+const Link = anchor`
+  display: flex;
+  font-size: 14px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-family: ${fonts.primary.fontFamily};
+  border-top: 1px solid #e5e5e5;
+  &:hover {
+    background-color: #f8f8f8;
+  }
+  ${props => (props.fadeIn ? fadeIn : null)}
+  ${props => (props.fadeOut ? fadeOut : null)}
+`
+
+const Avatar = styled.img`
+  padding: 10px 15px 10px 10px;
+`
+
+const FullWidthCol = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+`
+
+const Col = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+export interface State {
+  fadeIn: boolean
+  fadeOut: boolean
+  item_id: string
+}
+
+interface Props extends React.HTMLProps<HTMLAnchorElement> {
+  item?: any
+  id: string
+  _id: string
+  name: string
+  image_url
+}
+
+export default class ItemLink extends React.Component<Props, State> {
+  state = {
+    fadeIn: false,
+    fadeOut: false,
+    item_id: this.props.id,
+  }
+
+  followItem() {
+    return null
+  }
+
+  followItemFailed() {
+    null
+  }
+
+  onClick() {
+    this.followItem()
+  }
+
+  render() {
+    return (
+      <Link
+        onClick={this.onClick.bind(this)}
+        fadeIn={this.state.fadeIn}
+        fadeOut={this.state.fadeOut}
+        item_id={this.props.id}
+      >
+        <Col>{<Avatar src={this.props.image_url ? this.props.image_url : ""} width={50} height={50} />}</Col>
```

Comment:
What falsy values could `image_url` hold? If it’s `null`, `undefined`, or `""` then I wonder if you can’t just use those? If you really do need to default to an empty string, then a shorter version would be to do `this.props.image_url || ""`

Reply:
If my memory serves me correctly, I think the `image_url` can be `null`. I think I left this as an empty string because I thought there should be some sort of image place holder because that's the way search in force handles this. I planned to go back and add the image placeholder. Might be a good time to do that now 😄

Reply:
👍